pkg/envoy/accesslog_server.go

```diff
@@ -147,8 +147,10 @@ func (s *AccessLogServer) handleConn(ctx context.Context, conn *net.UnixConn) {
 			continue
 		}
 
-		flowdebug.Log(log.WithFields(logrus.Fields{}),
-			fmt.Sprintf("%s: Access log message: %s", pblog.PolicyName, pblog.String()))
+		flowdebug.Log(func() (*logrus.Entry, string) {
+			return log.WithFields(logrus.Fields{}),
```

Comment:
I think the call to `WithFields` is avoidable here. The `log` variable is a `*logrus.Entry` itself, so it can be returned as-is. Maybe the original intent was to convert a `*Logger` struct into an `*Entry` struct?

Reply:
hah, good catch!